client/client_test.go

```diff
@@ -1539,11 +1540,49 @@ func Test_Client_SetProxyURL(t *testing.T) {
 
 	app, dial, start := createHelperServer(t)
 	app.Get("/", func(c fiber.Ctx) error {
-		return c.SendString("hello world")
+		return c.SendString(c.Get("isProxy"))
 	})
 
 	go start()
 
+	fasthttpClient := &fasthttp.Client{
+		Dial:                     dial,
+		NoDefaultUserAgentHeader: true,
+		DisablePathNormalizing:   true,
+	}
+
+	// Create a simple proxy sever
+	proxyServer := fiber.New()
```

Comment:
This is probably creating a server in a port already used

Reply:
@efectn can you check this

Reply:
> Could we use a InmemoryListener for tests?

i dont't think we can, since FasthttpHTTPDialer only accepts a valid proxy url at ip:port format

However, i'll randomize de port instead of using fixed port